css/main.css

```diff
@@ -0,0 +1,627 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  padding: 15px;
+  color: white;
+  background-color: #3c3a39;
+  height: auto;
+  transition: all 1s ease-in-out 0.5s;
+}
+
+.logo-menubars {
+  min-width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.desktop-nav {
+  min-height: 100%;
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#logo {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.nav-link {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 20px;
+  color: #fff;
+  padding: 15px;
+  border: 1px solid transparent;
+  transition: 350ms;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars,
+.close-menubars {
+  cursor: pointer;
+}
+
+.close-menubars {
+  display: none;
+}
+
+.nav-link:hover {
+  border-bottom: 1px solid #d2d5db;
+}
+
+.close-menubars i,
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+#intro,
+#about-me {
+  background-color: #1c1a19;
+  color: white;
+  position: relative;
+}
+
+#intro {
+  padding: 10%;
+  padding-bottom: 20vh;
+}
+
+.intro-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+
+  /* height: 800px; */
```

Comment:
I kindly invite you to delete all the commented code, this will help to improve the code professionalism and to have a better and clean code. If you want to use code in the future, get help for Github project history. 😄